package.json

```diff
@@ -2,7 +2,7 @@
   "name": "snyk-vulnerability-scanner",
   "//": "Changing display name requires change in general.ts",
   "displayName": "Snyk Vulnerability Scanner",
-  "version": "1.0.7",
+  "version": "1.1.0",
```

Comment:
I think package-lock.json is missing here and runs out of sync.

Reply:
👍 had to resolve a conflict with master branh